lms/views/api/vitalsource.py

```diff
@@ -2,31 +2,35 @@
 
 from lms.security import Permissions
 from lms.services.exceptions import ProxyAPIError
-from lms.views.api.vitalsource_sample_data import book_data, toc_data
 
 
 @view_defaults(renderer="json", permission=Permissions.API)
 class VitalSourceAPIViews:
     def __init__(self, request):
         self.request = request
+        self.vitalsource_service = request.find_service(name="vitalsource")
 
     @view_config(route_name="vitalsource_api.books.info")
     def book_info(self):
         book_id = self.request.matchdict["book_id"]
 
-        # In future this will fetch data from VitalSource.
-        if book_id not in book_data:
+        book_info = self.vitalsource_service.book_info(book_id)
+
+        if not book_info:
             raise ProxyAPIError(f"Book {book_id} not found")
 
-        return book_data[book_id]
+        return {
+            "id": book_info["vbid"],
+            "title": book_info["title"],
+            "cover_image": book_info["resource_links"]["cover_image"],
```

Comment:
I'm not sure if a cover image is guaranteed to exist, but we'll get a Sentry error if it turns out that it doesn't.